lib/interceptor.js

```diff
@@ -50,7 +50,6 @@ var interceptor = {
         }
         if (init) {
           request.requestBody = init.body;
-          request.url = init.url;
```

Comment:
Why did you remove this?

Reply:
Oops, sorry, just saw your issue. Never mind then

Reply:
Are the changes made by @lacell75 acceptable for you here: https://github.com/chmanie/wdio-intercept-service/pull/56/files?

Reply:
Since `init.url` is an invalid option (see https://github.com/chmanie/wdio-intercept-service/issues/51) I don't see the need to use it.